### PR TITLE
Crossterm 0.11

### PR DESCRIPTION
**Pull request by TimonPost**
# 1

- [X] Updated crossterm Changelog 
- [x] Updated crossterm_cursor Changelog 
- [x] Updated crossterm_screen Changelog 
- [x] Updated crossterm_terminal Changelog 
- [x] Updated crossterm_style Changelog 
- [x] Updated crossterm_utils Changelog 
- [x] Updated crossterm_winapi Changelog 

# 2
- [X] Increased version number crossterm
- [X] Increased version number crossterm_cursor 
- [X] Increased version number crossterm_screen 
- [X] Increased version number crossterm_terminal 
- [X] Increased version number crossterm_style 
- [X] Increased version number crossterm_utils 
- [X] Increased version number crossterm_winapi 
- Updated version numbers sub dependencies
    Will happen when crossterm is going to be released, needs to be done while releasing step by step.

# 3
- [x] Update README crossterm
- [x] Update README crossterm_cursor 
- [x] Update README crossterm_screen 
- [x] Update README crossterm_terminal 
- [x] Update README crossterm_style 
- [x] Update README crossterm_utils 
- [x] Update README crossterm_winapi

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,71 @@
...
+
+[return-written-bytes]: https://github.com/TimonPost/crossterm/pull/212
+
+[return_crossterm_result]: https://github.com/TimonPost/crossterm/pull/232
+[added-key-event-tab]: https://github.com/TimonPost/crossterm/pull/239
+[added-key-event-enter]: https://github.com/TimonPost/crossterm/pull/236
+[fixed-get-set-terminal-size]: https://github.com/TimonPost/crossterm/pull/242
+
+# Changes crossterm 0.10.1
+
```
Comment (TimonPost) [inline docs/CHANGELOG.md]: This was more a kind of version patch, will put something here soon

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "crossterm"
-version = "0.10.1"
+version = "0.10.3"
```
Comment (zrzka) [inline Cargo.toml]: `0.11.0`?

Comment (TimonPost) [reply Cargo.toml]: Haha, yes

**Comment by TimonPost** [comment]
Going to do a rebase and then tomorrow we can plan the release

File: README.md [hunk 3]
```diff
@@ -114,7 +114,7 @@ Because it is more convenient, faster, and easier to use.
 ### Styled Text
 This module enables you to style the terminal text.
 
-Good documentation can be found at the following places: [docs](https://docs.rs/crossterm_style/), [book](https://timonpost.github.io/crossterm/docs/styling.html), [examples](https://github.com/TimonPost/crossterm/tree/master/examples/key_events.rs)
+Good documentation can be found at the following places: [docs](https://docs.rs/crossterm_style/), [book](https://timonpost.github.io/crossterm/docs/styling.html), [examples](https://github.com/crossterm-rs/crossterm/tree/master/examples/key_events.rs)
```
Comment (zrzka) [inline README.md]: I'd suggest to generate pages, setup them now and change these links as well. Pages are the only thing that are not redirected.

File: README.md [hunk 4]
```diff
@@ -237,7 +237,7 @@ terminal.write("Some text\n Some text on new line");
 ### Input Reading
 This module enables you to read user input events.
 
-Good documentation could be found on the following places: [docs](https://docs.rs/crossterm_input/), [book](https://timonpost.github.io/crossterm/docs/input.html), [examples](https://github.com/TimonPost/crossterm/tree/master/examples/key_events.rs)
+Good documentation could be found on the following places: [docs](https://docs.rs/crossterm_input/), [book](https://timonpost.github.io/crossterm/docs/input.html), [examples](https://github.com/crossterm-rs/crossterm/tree/master/examples/key_events.rs)
```
Comment (zrzka) [inline README.md]: I'd suggest to generate pages, setup them now and change these links as well. Pages are the only thing that are not redirected.

File: README.md [hunk 5]
```diff
@@ -294,7 +294,7 @@ input.disable_mouse_mode().unwrap();
 ```
 
 ### Alternate and Raw Screen
-These concepts are a little more complex and would take over the README, please check out the [docs](https://docs.rs/crossterm_screen/), [book](https://timonpost.github.io/crossterm/docs/screen.html), and [examples](https://github.com/TimonPost/crossterm/tree/master/examples).
+These concepts are a little more complex and would take over the README, please check out the [docs](https://docs.rs/crossterm_screen/), [book](https://timonpost.github.io/crossterm/docs/screen.html), and [examples](https://github.com/crossterm-rs/crossterm/tree/master/examples).
```
Comment (zrzka) [inline README.md]: I'd suggest to generate pages, setup them now and change these links as well. Pages are the only thing that are not redirected.

File: README.md [hunk 6]
```diff
@@ -55,7 +55,7 @@ This crate consists of five modules that are provided behind [feature flags](htt
 
 ## Getting Started
 
-This documentation is only for Crossterm version `^0.10`. If you have an older version of Crossterm, then I suggest you check the [Upgrade Manual](https://github.com/TimonPost/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/TimonPost/crossterm/tree/master/examples) folders with detailed examples for all functionality of this crate.
+This documentation is only for Crossterm version `^0.10`. If you have an older version of Crossterm, then I suggest you check the [Upgrade Manual](https://github.com/crossterm-rs/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/crossterm-rs/crossterm/tree/master/examples) folders with detailed examples for all functionality of this crate.
```
Comment (zrzka) [inline README.md]: I'd suggest to remove this whole sentence:

_This documentation is only for Crossterm version `^0.10`_

Just to lower the maintenance burden.

File: crossterm_input/README.md [hunk 7]
```diff
@@ -41,7 +41,7 @@ When you want to use other modules as well you might want to use crossterm with
 
 ## Getting Started
 
-This documentation is only for `crossterm_input` version `0.3` if you have an older version I suggest you check the [Upgrade Manual](https://github.com/TimonPost/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/TimonPost/crossterm/tree/master/crossterm_input/examples) folders with detailed examples for all functionalities of this crate.
+This documentation is only for `crossterm_input` version `0.3` if you have an older version I suggest you check the [Upgrade Manual](https://github.com/crossterm-rs/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/crossterm-rs/crossterm/tree/master/crossterm_input/examples) folders with detailed examples for all functionalities of this crate.
```
Comment (zrzka) [inline crossterm_input/README.md]: I'd suggest to remove following part of the sentence:

_This documentation is only for `crossterm_input` version `0.3`_

* Just to lower the maintenance burden
* Rewrite rest of the sentence in a way that have to consult upgrade manual if they're upgrading.

File: crossterm_screen/Cargo.toml [hunk 8]
```diff
@@ -12,8 +12,8 @@ readme = "README.md"
...
 [dependencies]
-crossterm_utils = { path="../crossterm_utils", version = "0.2.4"}
+crossterm_utils = { path="../crossterm_utils", version = "0.3.0"}
 
 [target.'cfg(windows)'.dependencies]
-winapi = { version =  "0.3.7", features = ["minwindef", "wincon"] }
-crossterm_winapi = { path="../crossterm_winapi", version = "0.1.5"}
\ No newline at end of file
+winapi = { version =  "0.3.8", features = ["minwindef", "wincon"] }
+crossterm_winapi = { path="../crossterm_winapi", version = "0.2.0"}
```
Comment (zrzka) [inline crossterm_screen/Cargo.toml]: New line at the end please :)

File: crossterm_screen/README.md [hunk 9]
```diff
@@ -44,7 +44,7 @@ In case you are wondering what 'alternate' or 'raw' screen is, you could checkou
 ## Getting Started
 
 This documentation is only for `crossterm_screen` version `0.2`.
```
Comment (zrzka) [inline crossterm_screen/README.md]: Same as above comments, should be removed.

File: crossterm_style/README.md [hunk 10]
```diff
@@ -41,7 +41,7 @@ When you want to use other modules as well you might want to use crossterm with
 
 ## Getting Started
 
-This documentation is only for `crossterm_style` version `0.3` if you have an older version I suggest you check the [Upgrade Manual](https://github.com/TimonPost/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/TimonPost/crossterm/tree/master/crossterm_style/examples) folders with detailed examples for all functionality of this crate.
+This documentation is only for `crossterm_style` version `0.3` if you have an older version I suggest you check the [Upgrade Manual](https://github.com/crossterm-rs/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/crossterm-rs/crossterm/tree/master/crossterm_style/examples) folders with detailed examples for all functionality of this crate.
```
Comment (zrzka) [inline crossterm_style/README.md]: Same as above comments, we shouldn't include the version here.

File: crossterm_terminal/CHANGELOG.md [hunk 11]
```diff
@@ -1,8 +1,21 @@
-# Changes crossterm_style 0.2.2
+# Changes crossterm_style 0.3.0
```
Comment (zrzka) [inline crossterm_terminal/CHANGELOG.md]: crossterm_style -> crossterm_terminal

File: crossterm_terminal/README.md [hunk 12]
```diff
@@ -41,7 +41,7 @@ When you want to use other modules as well you might want to use crossterm with
 
 ## Getting Started
 
-This documentation is only for `crossterm_terminal` version `0.2` check the [examples](https://github.com/TimonPost/crossterm/tree/master/examples) folders with detailed examples for all functionality of this crate.
+This documentation is only for `crossterm_terminal` version `0.2` check the [examples](https://github.com/crossterm-rs/crossterm/tree/master/examples) folders with detailed examples for all functionality of this crate.
```
Comment (zrzka) [inline crossterm_terminal/README.md]: Same as above comments, version included.

File: docs/CHANGELOG.md [hunk 13]
```diff
@@ -1,24 +1,92 @@
-# Changes crossterm 0.10.0
-- Implemented command API, to have better performance and more control over how and when commands are executed. [PR](https://github.com/TimonPost/crossterm/commit/1a60924abd462ab169b6706aab68f4cca31d7bc2), [issue](https://github.com/TimonPost/crossterm/issues/171)
+# Changes crossterm 0.11.0
+
```
Comment (zrzka) [inline docs/CHANGELOG.md]: I'd include something like that we moved it to separate organization?

File: crossterm_winapi/Cargo.toml [hunk 14]
```diff
@@ -1,9 +1,9 @@
 [package]
 name = "crossterm_winapi"
-version = "0.1.5"
+version = "0.2.0"
 authors = ["T. Post"]
 description = "An WinApi wrapper that provides some basic simple abstractions aground common WinApi calls"
-repository = "https://github.com/TimonPost/crossterm_winapi"
+repository = "https://github.com/crossterm-rs/crossterm_winapi"
```
Comment (zrzka) [inline crossterm_winapi/Cargo.toml]: https://github.com/crossterm-rs/crossterm_winapi -> https://github.com/crossterm-rs/crossterm

**Comment by zrzka** [comment]
Travis CI should be reenabled again. You can do it as an owner. Maybe you'll need to rescan repositories (on Travis CI) to see it.

**Comment by zrzka** [comment]
Make the logo a bit smaller, it's quite big for the README :)

Screen Shot 2019-09-24 at 15 52 41

File: README.md [hunk 15]
```diff
@@ -1,5 +1,7 @@
-# Crossterm | cross-platform terminal manipulating library.
- [![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=Z8QK6XU749JB2) ![Travis][s7] ![Workflow][s8] [![Latest Version][s1]][l1] [![MIT][s2]][l2] [![docs][s3]][l3] ![Lines of Code][s6] [![Join us on Discord][s5]][l5]
+<img align="center" style="margin-bottom=10px; height=255px" src="docs/crossterm_full.png" />
```
Comment (zrzka) [inline README.md]: Use the full URL here. Two phases:

* use the `0.11` branch to test it, how it looks like, etc.
* then, just before merging, replace it with the `master` branch

Reason? Same README.md is used on [crates.io](https://crates.io/crates/crossterm) and there's nothing like `docs/crossterm_full.png`.

**Pull request merged**